spec/features/conditions/user_sees_conditions_for_id_spec.rb

```diff
@@ -0,0 +1,18 @@
+require_relative '../../spec_helper'
+
+describe "Conditions/id" do
+  describe "when a user visits /conditions/id" do
+    it "they see a welcome message with the date" do
+      condition = Condition.create(date: '12/01/2000', precipitation_inches: 5,
+                                   max_temperature_f: 50, min_temperature_f: 20,
+                                   mean_temperature_f: 35, mean_visibility_miles: 10,
+                                   mean_wind_speed_mph: 20, mean_humidity: 20,
+                                   zip_code: 94107)
+      visit('/conditions/1')
```

Comment:
how do we know that the condition you created on line 6 has an id of 1? how can we make this more flexible?

`visit("/conditions/#{condition.id}")`